src/styles/components/_Header.scss

```diff
@@ -117,11 +117,8 @@ Homepage header
 @media (max-width: $width-sm) {
   // On small screens, the logo padding was creating a wonky focus style
   .hc-c-header {
-    .ds-l-row {
-      margin-left: 0;
-    }
-
     .hc-c-logo-link {
+      margin-left: 8px;
```

Comment:
Adding margin instead of padding puts the spacing outside the link element instead of inside and pulls the focus in